scripts/pr-merge-autopilot.sh

```
@@ -0,0 +1,497 @@
+    if [[ "$DRY_RUN" == "false" ]]; then
+      # Post reply via GraphQL
+      REPLY_BODY="Acknowledged — addressed or accepted as advisory."
+      REPLY_MUTATION='
+mutation($threadId:ID!, $body:String!) {
+  addPullRequestReviewComment(input:{pullRequestReviewThreadId:$threadId, body:$body}) {
+    comment {
+      id
+    }
+  }
+}
+'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)